CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Add Dockerfile template and entry.sh for running the generated tar as a Docker container [Praneeth]
```

Comment:
The convention elsewhere seems to be to make a separate commit for changleog updates.